demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -54,6 +59,51 @@
 
 DEFAULT_SENTENCES = ['getting started and learn how to build an integration']
 
+logger = logging.getLogger('demisto-sdk')
+
+
+@lru_cache(None)
+def is_docker_available():
+    try:
+        docker_client: docker.DockerClient = init_global_docker_client(log_prompt='LintManager')
+        docker_client.ping()
+        return True
+    except Exception:
+        return False
+
+
+@lru_cache(None)
+def are_modules_installed_for_verify(content_path: str) -> bool:
+    """ Check the following:
+        1. npm packages installed - see packs var for specific pack details.
+        2. node interperter exists.
+    Returns:
+        bool: True If all req ok else False
+    """
+    missing_module = []
+    valid = True
+    # Check node exist
+    stdout, stderr, exit_code = run_command_os('node -v', cwd=content_path)
+    if exit_code:
+        print_warning(f'There is no node installed on the machine, Test Skipped, error - {stderr}, {stdout}')
+        valid = False
+    else:
+        # Check npm modules exsits
+        stdout, stderr, exit_code = run_command_os(f'npm ls --json {" ".join(REQUIRED_MDX_PACKS)}',
+                                                   cwd=content_path)
+        if exit_code:  # all are missinig
+            missing_module.extend(REQUIRED_MDX_PACKS)
+        else:
+            deps = json.loads(stdout).get('dependencies', {})
+            for pack in REQUIRED_MDX_PACKS:
+                if pack not in deps:
+                    missing_module.append(pack)
+    if missing_module:
+        valid = False
+        print_warning(f"The npm modules: {missing_module} are not installed. Use "
```

Comment:
may be more than one, i'd add 
```python
missing_modules_str = ', '.join(sortred(missing_module))
```

Reply:
prints as list, that should be built in (also I didnt write this func, only moved it)